### PR TITLE
Update hub-install.adoc

### PR DESCRIPTION
**Pull request by chiaradiamarcelo**
# Description

Update the documentation regarding the Hub XMLRPC installation, according to the latest changes on it.

# Target branches

To what product version this document applies?

- [ ] Manager-3.2
- [ ] Manager-4.0
- [x] Manager-4.1
- [x] Uyuni

# Links

Fixes https://github.com/SUSE/spacewalk/issues/11739

### CODE REVIEW

File: modules/large-deployments/pages/hub-install.adoc [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 
 
 
 .Procedure: Installing and Configuring the XMLRPC API
 . On the Hub Server, or on a host that has access to all peripheral servers' XMLRPC APIs, install the ``hub-xmlrpc-api`` package.
 The package is available in the {productname} {productnumber} repositories.
-. Verify if the configuration file available at  ``/etc/hub/hub-xmlrpc-api-config.json`` is pointing to the correct Hub instance and modify it as required.
-. OPTIONAL: In the configuration file, check the settings for the timeout parameters ``connect_timeout`` and ``read_write_timeout``.
-The values for these parameters are defined in seconds.
+* If the ``hub-xmlrpc-api`` package is installed on the Hub then no further configuration is necessary. The daemon can be started via systemd as a ``hub-xmlrpc-api.service`` file is provided by the package.
```
Comment (Loquacity) [inline modules/large-deployments/pages/hub-install.adoc:11]:
```suggestion
. If the ``hub-xmlrpc-api`` package is installed on the Hub then no further configuration is necessary and you can start the service using systemd.
The ``hub-xmlrpc-api.service`` file is provided by the package.
```

File: modules/large-deployments/pages/hub-install.adoc [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
+* If the ``hub-xmlrpc-api`` package is installed on a separate host then verify if ``HUB_API_URL`` field defined in the configuration file available at ``/etc/hub/hub.conf`` is pointing to the correct Hub instance and modify it as required.
```
Comment (Loquacity) [inline modules/large-deployments/pages/hub-install.adoc:12]:
```suggestion
Alternatively, if the ``hub-xmlrpc-api`` package is installed on a separate host, you need to verify that the configuration file is pointing to the correct Hub instance.
Open the [path]``/etc/hub/hub.conf`` configuration file and check that the ``HUB_API_URL`` field is correct.
```

File: modules/large-deployments/pages/hub-install.adoc [hunk 3]
```diff
@@ -1,14 +1,18 @@
...
+. OPTIONAL: In the configuration file, check the settings for the timeout parameters ``HUB_CONNECT_TIMEOUT`` and ``HUB_REQUEST_TIMEOUT``.
+* ``HUB_CONNECT_TIMEOUT`` refers to the maximum number of seconds to wait for a response when connecting to a Server.
```
Comment (Loquacity) [inline modules/large-deployments/pages/hub-install.adoc:14]:
```suggestion
* ``HUB_CONNECT_TIMEOUT`` refers to the maximum number of seconds to wait for a response when connecting to a server.
```

File: modules/large-deployments/pages/hub-install.adoc [hunk 4]
```diff
@@ -1,14 +1,18 @@
...
+* ``HUB_REQUEST_TIMEOUT`` refers to the maximum numbr of seconds to wait for a response when calling a Server method.
```
Comment (Loquacity) [inline modules/large-deployments/pages/hub-install.adoc:15]:
```suggestion
* ``HUB_REQUEST_TIMEOUT`` refers to the maximum number of seconds to wait for a response when calling a server method.
```

File: modules/large-deployments/pages/hub-install.adoc [hunk 5]
```diff
@@ -1,14 +1,18 @@
...
+
+The default values for these parameters should suffice in most settings.
```
Comment (Loquacity) [inline modules/large-deployments/pages/hub-install.adoc:17]:
```suggestion
In most cases, these parameters can be left at their default values.
```

**Comment by chiaradiamarcelo** [comment]
Great suggestions, thankss @Loquacity !

**Pull request merged**